vita3k/renderer/include/renderer/state.h

```diff
@@ -80,7 +81,7 @@ struct State {
 
     bool need_page_table = false;
 
-    virtual bool init(const char *shared_path, const bool hashless_texture_cache) = 0;
+    virtual bool init(const fs::path static_assets, const bool hashless_texture_cache) = 0;
```

Comment:
```suggestion
    virtual bool init(const fs::path &static_assets, const bool hashless_texture_cache) = 0;
```
Always use a reference when passing a const fs::path (or std::string)